content/sensu-go/5.14/release-notes.md

```diff
@@ -40,6 +41,42 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.14.0 release notes
+
+**October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that enable round robin schedule recovery after quorum loss and enhanced TLS authentication support.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.14.0.
+
+**NEW FEATURES:**
+
+- The [web UI][80] now includes an error dialog option that allows users to wipe the application's persisted state (rather than having to manually wipe their local/session storage). This can help in the rare cases that something in the persisted state is leading to an uncaught exception.
+- The [web UI][80] now respects the system preference for operating systems with support for selecting a preferred light or dark theme.
+- `sensuctl dump` can now list the types of supported resources with `sensuctl dump --types`.
+- The [entity resource][82] now includes the `sensu_agent_version` field, which reflects the Sensu Semantic Versioning (SemVer) version of the agent entity.
+- There are two new [advanced configuration options][83] for `sensu-backend` using embedded etcd: `etcd-heartbeat-interval` and `etcd-election-timeout`.
+
+**IMPROVEMENTS:**
+
+- ([Licensed tier][79]) Added support for mutual TLS authentication between agents and backends.
+- ([Licensed tier][79]) Added support for CRL URLs for mTLS authentication.
+- ([Licensed tier][79]) Support agent [TLS authentication][81] is usable with a licensed sensu-backend.
+- To facilitate discussion about feature requests, in the Web UI, feedback is directed to Discourse rather than the GitHub repository's issues page.
+- In the [web UI][80], when a user lands on a page inside a namespace that no longer exists or they do not have access to, the drawer opens to that namespace switcher to help clarify next steps.
+- Updated Go version from 1.12.3 to 1.13.1.
+
+**FIXES:**
+
+- ([Licensed tier][79]) `sensuctl` can now create federation resources.
+- ([Licensed tier][79]) `sensuctl` on Windows can now create Postgres resources.
+- ([Licensed tier][79]) Fixed a bug that resulted in event metrics being ignored when using the Postgres store.
+- Splayed proxy checks are executed every interval (instead of every interval + interval * splay_coverage).
```

Comment:
```suggestion
- Fixed a bug that caused checks to stop executing after a network error.
- Splayed proxy checks are executed every interval (instead of every interval + interval * splay_coverage).
```